src/delivery/tests.py

```diff
@@ -264,16 +264,17 @@ def test_save_route_and_retrieve(self):
                                     content_type="application/json")
         response = self.client.get(
             '/delivery/getDailyOrders/?route=' +
-            str(self.route_id) + '&mode=retrieve')
+            str(self.route_id) + '&if_exist_then_retrieve=true')
         self.assertTrue(b'Dallaire' in response.content)
```

Comment:
This does not really assure of a proper order right? It only says that this member is in the route.